packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -314,7 +311,6 @@ function updateMode(
     nextChildren,
     renderExpirationTime,
   );
-  memoizeProps(workInProgress, nextChildren);
```

Comment:
This one is interesting because it doesn't memoize its props. Only its children. This is a bug since it won't bail out ever.